xcode.md

```diff
@@ -65,13 +65,15 @@ Got access to old versions of Apple installers you know are legit? Submit some h
 | [XCode 5.1.1 (10.8)][5.1.1]       | `e4bb45174324c3a4b7c66fa1db1083ccbbe2334e` | xcode_5.1.1.dmg
 | [XCode 5.0.2 (10.8)][5.0.2]       | `de23e3f4644dbb434ed862a3ce002e2de555f6f2` | xcode_5.0.2.dmg
 | [XCode 4.6.3 (10.7-10.8)][4.6.3]  | `f8f9d8a3eec9c46072c02e0007f6abe411e674f8` | xcode4630916281a.dmg
+| [XCode 4.6 (10.7-10.8)][4.6]      | `e677cd170fd5090ea0670bb31cb2023b58a9d323` | xcode460417218a.dmg
 | [XCode 4.5.2 (10.7-10.8)][4.5.2]  | `2dea0b49f9f35b91ad2abc7f3b71889679febac1` | xcode4520418508a.dmg
 | [XCode 4.4.1 (10.7)][4.4.1]       | `4b8e927c7c58885fe91a36a21952604285b8d60f` | xcode_4.4.1_6938145.dmg
 | [XCode 4.3.2 (10.7)][4.3.2]       | `8e2723f24f2a58af78317c115e1dc8e4f3c96b43` | xcode_432_lion.dmg
 | [XCode 4.2 (10.6-10.7)][4.2]      | `de748f5d096c55666b3ac22ee6fbe0e512206b36` | installxcode_42_lion.dmg
 | [XCode 4.1 (10.6-10.7)][4.1]      | `7ccb636d51d54e88c2d27be55cbff107d5be96a0` | installxcode_41_lion.dmg
 | [XCode 4.0.2 (10.6)][4.0.2]       | `684265b6310566c5b0080b1a47e5ddc2c680c929` | xcode_4.0.2_and_ios_sdk_4.3.dmg
 | [XCode 3.2.6 (10.6)][3.2.6]       | `9eef71cd6687d19f42ce16e8b60fa3b549a58941` | xcode_3.2.6_and_ios_sdk_4.3.dmg
+| [XCode 3.2.5 (10.6)][3.2.5]       | `4332db9c9a1f4d85f65d11476bd941b8e1d3fc98` | xcode_3.2.5_and_ios_sdk_4.2_final.dmg
```

Comment:
```
$ sha1sum xcode_3.2.5_and_ios_sdk_4.2_final.dmg 
4332db9c9a1f4d85f65d11476bd941b8e1d3fc98  xcode_3.2.5_and_ios_sdk_4.2_final.dmg
```

this is an old archived file, but untouched. could have bitrotted.. I cant really re-fetch the file to verify at the moment due to broadband constraints, so I'll update the PR and just remove the 3.2.5 one for now